client/components/ReportRerun/RerunDashboard.jsx

```diff
@@ -79,19 +79,14 @@ const RerunDashboard = ({ activeRuns, onRerunClick }) => {
                         : 'earlier versions'}{' '}
                       of {run.botName} can be automatically updated with the
                       &quot;Start Generating Reports&quot; button below.
-                      {/* TODO: Remove below when https://github.com/w3c/aria-at-app/issues/1417 is addressed */}
-                      <br />
-                      <br />
-                      Note: The button is temporarily disabled while we work on
-                      upcoming feature updates.
                     </p>
                   </div>
 
                   <div className={styles.actionHeader}>
                     {/* TODO: Re-enable when https://github.com/w3c/aria-at-app/issues/1417 is addressed */}
```

Comment:
This comment can go, too.

Reply:
[99030f7](https://github.com/w3c/aria-at-app/pull/1459/commits/99030f7dfcd35e6815074ff1ca8ee1ef67bd5d98)